pkg/settlement/pseudosettle/pseudosettle.go

```diff
@@ -90,6 +94,9 @@ func (s *Service) Pay(ctx context.Context, peer swarm.Address, amount uint64) er
 	}()
 
 	s.logger.Tracef("sending payment message to peer %v of %d", peer, amount)
+
+	s.metrics.TotalSentPseudoSettlements.Add(float64(amount))
```

Comment:
this should be after the `WriteMsgWithContext`, so it does not increase if sending fails